helmfile.d/stacks/crossplane.yaml.gotmpl

```diff
@@ -0,0 +1,44 @@
+---
+templates:
+  crossplane:
+    condition: ck8sAnyCluster.enabled
+    installed: {{ and (.Values | get "ck8sAnyCluster.enabled" false) (.Values | get "crossplane.enabled" false) }}
```

Comment:
`installed` cannot be on a shared template if any of those inheriting it has their own `installed`

"falsy" values are skipped in the merge within helmfile.

Reply:
Good to know! https://github.com/elastisys/compliantkubernetes-apps/pull/2779/commits/6d306c7bee7eda428086f16f17bba2de90426b17